### DISCUSSION TITLE
What task do you need to run to copy js files under web-src?

### DISCUSSION THREAD

**Discussion by dburner** [dc-js/dc.js#1835] [Q&A]
I want to create a new custom chart. In order to do that I want to start from an example, so I copied an example html under `web-src/examples` but I can not find any task that will build/copy necessary files to `web-src/js`

**Comment by gordonwoodhull** [comment]
Hi @dburner! As its name suggests,  `web-src` is the source for the `web` directory which gets created or updated during the build.  That copying happens here:

https://github.com/dc-js/dc.js/blob/4078c0739b5e83cdbf7624c5e3546e5d7e672675/Gruntfile.js#L191-L195

Currently all the examples except for the main stock example have their js inside the html. However all artifacts from `dist/` and all dependency libraries from `node_modules` are copied in this task a few lines down:

https://github.com/dc-js/dc.js/blob/4078c0739b5e83cdbf7624c5e3546e5d7e672675/Gruntfile.js#L211-L225

Hope this answers your question!
